components/results_table/commons/results_table.lua

```diff
@@ -82,7 +83,10 @@ function ResultsTable:buildRow(placement)
 			placement.pagename
 		))
 
-	if self.config.playerResultsOfTeam or self.config.queryType ~= Opponent.team then
+	if self.config.playerResultsOfTeam or
+		self.config.queryType ~= Opponent.team or
+		Table.isNotEmpty(self.config.aliases) then
+
```

Comment:
IMO an option to hide the team icon for non-aliased ones would be great.

Reply:
for teams the team column will only show if you have aliases
also imo  showing the icon for some but not others imo looks weird

Reply:
it would only show for these that aren't the main name

Reply:
like this: https://liquipedia.net/simracing/User:Hesketh2/Test#with_doNotShowNonAliased.3Dtrue
(as comparison the one above shows the icon always)
https://liquipedia.net/commons/index.php?title=Module%3AResultsTable%2FBase%2Fdev&type=revision&diff=478628&oldid=478466

Reply:
You are right, when using logos it is fine. The current version on AoE would just use name without logo, and it looked weird having the same name over and over again.